app/src/main/java/com/twilio/conversations/app/manager/MessageListManager.kt

```diff
@@ -74,23 +74,25 @@ class MessageListManagerImpl(
             uuid
         )
         conversationsRepository.insertMessage(message)
-        val sentMessage = conversation.sendMessage(options).toMessageDataItem(identity, uuid)
+
+        val unsentMessage = conversation.prepareMessage().setBody(text).setAttributes(attributes).build()
```

Comment:
Why don't you use here `buildAndSend`?

Reply:
Yes, could use `buildAndSend`.
However, I've added `UnsentMessage.sendMessage()` extension, I think thus code is more clear and clean.

Reply:
Now using buildAndSend()